src/libraries/Microsoft.Bcl.Memory/ref/Microsoft.Bcl.Memory.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I understand that you follow the pattern from existing Bcl projects but I wonder why we need this reference source project and compile it, given that we don't ship it in the package. That seems like unnecessary work to me.

Reply:
Seems reasonable to support a case where we don't have the ref project - but we'd want to make sure nothing else in the build system depends on it.

Reply:
Let's follow-up on that in a separate issue.